ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.3-ocp4.16-lp-rosa-classic.yaml

```diff
@@ -0,0 +1,92 @@
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  mtc-python-client:
+    name: mtc-python-client
+    namespace: mtc-qe
+    tag: master
+  oadp-apps-deployer:
+    name: oadp-apps-deployer
+    namespace: oadp-qe
+    tag: master
+  oadp-e2e-qe:
+    name: oadp-e2e-qe
+    namespace: oadp-qe
+    tag: master
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/testing/interop/Dockerfile
+  inputs:
+    mtc-python-client:
+      paths:
+      - destination_dir: mtc-python-client.tar.gz
+        source_path: /mtc-python-client.tar.gz
+    oadp-apps-deployer:
+      paths:
+      - destination_dir: oadp-apps-deployer.tar.gz
+        source_path: /oadp-apps-deployer.tar.gz
+    oadp-e2e-qe:
+      paths:
+      - destination_dir: oadp-e2e-qe.tar.gz
+        source_path: /oadp-e2e-qe.tar.gz
+  to: oadp-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: oadp-rosa-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "oadp-s3-*", "failure_type": "all", "classification":  "Infrastructure Provisioning - S3 Buckets", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "oadp-execute-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 3}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "oadp-execute-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "OADP", "group": {"name": "lp-tests", "priority": 3}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/rosa/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.16-lp","rosa-classic-lp","oadp-lp"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.16"
+      OPERATORS: |
+        [
+          {"name": "redhat-oadp-operator", "source": "redhat-operators", "channel": "stable-1.4", "install_namespace": "openshift-adp", "target_namespaces": "openshift-adp", "operator_group":"oadp-operator-group"}
+        ]
+    test:
+    - chain: install-operators
+    - ref: oadp-execute-tests
+    workflow: oadp-rosa-aws-sts
+zz_generated_metadata:
+  branch: main
+  org: oadp-qe
+  repo: oadp-qe-automation
+  variant: oadp1.3-ocp4.16-lp-rosa-classic
```

Comment:
here again change the step name.

Reply:
also where is can see what suite your are running ?

we need to run e2e/cloudstorage without focus 

@PrasadJoshi12 please confirm

Reply:
@tareqalayan If cluster has an OIDC endpoint then our script sets the test folder path to e2e/cloudstorage. 
https://gitlab.cee.redhat.com/app-mig/oadp-e2e-qe/-/blob/master/test_settings/scripts/test_runner.sh?ref_type=heads#L134

Reply:
we can see it here as well
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/54603/rehearse-54603-periodic-ci-oadp-qe-oadp-qe-automation-main-oadp1.3-ocp4.16-lp-rosa-classic-oadp-rosa-aws/1815261799100977152/artifacts/oadp-rosa-aws/oadp-execute-tests/build-log.txt

```
+ ginkgo run -p --nodes=1 -mod=mod e2e/cloudstorage/ --show-node-events -- -credentials_file=/tmp/test-settings/credentials -vsl_credentials_file=/tmp/test-settings/aws_vsl_creds -oadp_namespace=openshift-adp -settings=/alabama/cspi/output_files/api-ci-rosa-s-olq3-2wzs-s1-devshift-org:6443/settings.json -must_gather_image=registry.redhat.io/oadp/oadp-mustgather-rhel9:1.3.2 -timeout_multiplier=2 -skip_dpa_creation=false --ginkgo.junit-report=/logs/artifacts/junit_oadp_interop_results.xml '--ginkgo.label-filter=!// || (// && !exclude_aws && (!/target/ || target_aws) )  ' --ginkgo.focus=interop

```

Reply:
Will be excluded from this PR and resolved in [Update OADP, Serverless and AMQ-Broker on ROSA and expand to OCP 4.16](https://github.com/openshift/release/commit/abd4635e57961ddf4a7310bd3677f5dbc471a08a)
Please review there, thank you very much